test/integration/node-specific/client_close.test.ts

```diff
@@ -0,0 +1,372 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+
+import { type TestConfiguration } from '../../tools/runner/config';
+import { runScriptAndGetProcessInfo } from './resource_tracking_script_builder';
+
+describe.skip('MongoClient.close() Integration', () => {
+  // note: these tests are set-up in accordance of the resource ownership tree
+
+  let config: TestConfiguration;
+
+  beforeEach(function () {
+    config = this.configuration;
+  });
+
+  describe('Node.js resource: TLS File read', () => {
+    describe('when client is connecting and reads an infinite TLS file', () => {
+      it('the file read is interrupted by client.close()', async function () {
+        await runScriptAndGetProcessInfo(
+          'tls-file-read',
+          config,
+          async function run({ MongoClient, uri, log, chai }) {
+            const infiniteFile = '/dev/zero';
+            const client = new MongoClient(uri, { tlsCertificateKeyFile: infiniteFile });
+            client.connect();
+            log({ ActiveResources: process.getActiveResourcesInfo() });
```

Comment:
Do we use the `ActiveResource` logs anywhere?  If not, can they be removed?

Reply:
No they're for debugging purposes.